ethereum/contracts/relayer/wormholeRelayer/WormholeRelayerGovernance.sol

```diff
@@ -93,7 +95,11 @@ abstract contract WormholeRelayerGovernance is WormholeRelayerBase, ERC1967Upgra
             foreignAddress;
     }
 
+    event ContractUpgraded(address indexed oldContract, address indexed newContract);
```

Comment:
I believe ERC1967Upgrade already emits a similar event

Reply:
yup - this is the same event from TokenBridge, NFTBridge, CoreBridge, etc for consistency